extension/background.js

```diff
@@ -16,39 +16,39 @@ chrome.runtime.onInstalled.addListener(function() {
 });
 
 chrome.contextMenus.onClicked.addListener(function(info, tab) {
-	toggleUI(tab);
+    toggleUI(tab);
 })
 
 chrome.runtime.onInstalled.addListener(function() {
 
-	// Redirect user to a demo page, and auto-start the app there.
-	// TODO: Is this a good page for users to play around w/ the app?
-	let demoUrl = 'https://en.wikipedia.org/wiki/Language';
+    // Redirect user to a demo page, and auto-start the app there.
+	let demoUrl = 'https://en.wikipedia.org/wiki/Reading';
 	chrome.tabs.create({
 	    url: demoUrl,
 	    active: true
 	}, function(tab) {
-		chrome.tabs.sendMessage(tab.id, {command: "toggleUI"}, function(response) {});
+        // Have to wait for new tab to load; see: https://stackoverflow.com/questions/5816373/chrome-tab-create-and-tab-getselected
+        /* TODO: Change this from this obv super hacky strategy. 
+        Problem is: Only want to sendMessage once, AFTER the new tab is loaded. If do nothing, it sends the message to old page, 
```

Comment:
one way to do this: send a message back from content script to background, that's how u know it's loaded
**pretty sure u can do this, but havent tried.

Reply:
TODO: is fine.

Reply:
Option 2: save in settings "have seen tutorial?"
This has the advantage that you can control tutorial completeness check.
e.g. keep reshowing tutorial until user has completed a task.

Reply:
O but leave this in next CL, cos this is big enough.

Reply:
I love option 2; I'm not sure that will help the "wait for new tab to be open" problem though? Adding TODO either way